src/ui_basic/panel.cc

```diff
@@ -206,9 +224,15 @@ int Panel::do_run() {
 		if (start_time >= next_draw_time) {
 			RenderTarget& rt = *g_gr->get_render_target();
 			forefather->do_draw(rt);
-			rt.blit((app->get_mouse_position() - Vector2i(3, 7)),
-			        app->is_mouse_pressed() ? default_cursor_click_ : default_cursor_);
-
+			if (get_config_bool("sdl_cursor", false)) {
```

Comment:
is get_config...() fast enough to check this every time?

Reply:
I measured ~6 microseconds on my PC. And the SDL calls take ~30 microseconds. Yeah, probably worth caching.